Cargo.toml

```diff
@@ -29,12 +29,13 @@ error-chain = { version = "0.12.4", default-features = false }
 reqwest = { version = "0.11.4", features = ["blocking", "json"] }
 tungstenite = { version = "0.17.3", features = ["native-tls"] }
 url = "2.2.2"
+tokio = { version = "1.21.2", features = ["full"] }
```

Comment:
is this a library dependency, or should be a dev depedency only?

Reply:
ok, you're right. fixed it